src/main/resources/component-biz.conf

```
@@ -1325,6 +1329,13 @@ security {
 
     profiles {
 
+        logged-in {
```

```suggestion
        flag-logged-in {
```
`logged-in` is only a mapped permission name in S2, the sirius generic one is `flag-logged-in`